build.gradle

```diff
@@ -14,127 +14,83 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 buildscript {
   ext {
+    korkVersion = "5.2.6"
+    fiatVersion = "1.0.0"
     kotlinVersion = "1.3.10"
     junitPlatformVersion = "1.0.2"
   }
+
   repositories {
     jcenter()
     maven { url "https://spinnaker.bintray.com/gradle" }
     maven { url "https://plugins.gradle.org/m2/" }
   }
 
   dependencies {
-    classpath 'com.netflix.spinnaker.gradle:spinnaker-dev-plugin:5.2.1'
-    classpath "org.junit.platform:junit-platform-gradle-plugin:${junitPlatformVersion}"
+    classpath 'com.netflix.spinnaker.gradle:spinnaker-dev-plugin:6.0.0'
+    if (Boolean.valueOf(enablePublishing)) {
+      classpath "com.netflix.spinnaker.gradle:spinnaker-gradle-project:6.0.0"
+    }
     classpath "com.netflix.nebula:nebula-kotlin-plugin:$kotlinVersion"
+    classpath "org.junit.platform:junit-platform-gradle-plugin:${junitPlatformVersion}"
+
   }
 }
 
 allprojects {
   group = "com.netflix.spinnaker.clouddriver"
   apply plugin: 'spinnaker.base-project'
-  apply plugin: 'groovy'
-
-  ext {
-    spinnakerDependenciesVersion = '1.44.1'
-    if (project.hasProperty('spinnakerDependenciesVersion')) {
-      spinnakerDependenciesVersion = project.property('spinnakerDependenciesVersion')
-    }
+  if (Boolean.valueOf(enablePublishing)) {
+    apply plugin: 'spinnaker.project'
   }
+  apply plugin: 'java-library'
+  apply plugin: 'groovy'
+  apply plugin: 'nebula.kotlin'
 
-  def checkLocalVersions = [spinnakerDependenciesVersion: spinnakerDependenciesVersion]
-  if (ext.has('versions')) {
-    def extVers = ext.get('versions')
-    if (extVers instanceof Map) {
-      checkLocalVersions.putAll(extVers)
-    }
-  }
+  sourceSets.main.java.srcDirs = []
+  sourceSets.main.groovy.srcDirs += ["src/main/java"]
 
-  def localVersions = checkLocalVersions.findAll { it.value.endsWith('-SNAPSHOT') }
```

Comment:
we should prob keep mavenlocal support, see https://github.com/spinnaker/orca/blob/master/build.gradle#L56